lib/rubocop/cop/layout/block_alignment.rb

```diff
@@ -51,16 +52,17 @@ module Layout
       #   # bad
       #
       #   foo.bar
-      #      .each do
-      #        baz
-      #          end
+      #     .each do
+      #       baz
+      #         end
       #
       #   # good
       #
       #   foo.bar
       #     .each do
-      #        baz
```

Comment:
[memo] This indentation also changes at `.each do` with bad/good, which seemed confusing, so we are fixing that.